packages/plugins/src/cli/install/index.ts

```diff
@@ -14,17 +15,63 @@ export const handleInstall = async () => {
 
   const dependencyInstall = await spawnAsync('forge', args)
   if (dependencyInstall.code !== 0) {
-    // The `stdout` contains the trace of the error.
-    console.log(dependencyInstall.stdout)
-    // The `stderr` contains the error message.
-    console.log(dependencyInstall.stderr)
-    process.exit(1)
+    // If installation fails due to missing tag, then run forge update and attempt again
+    if (
+      dependencyInstall.stderr.includes(
+        `Tag: "${CONTRACTS_LIBRARY_VERSION}" not found for repo "https://github.com/sphinx-labs/sphinx"`
+      )
+    ) {
+      // Run forge install since installing with the incorrect version deletes the previous installation
+      const install = await spawnAsync('forge', ['install'])
```

Comment:
Running `forge install` without any arguments installs all existing dependencies, not just Sphinx. Is that the intended effect? If not, would it make more sense to run this command with our package as an argument?

Reply:
That is the intended effect.